criu/bpfmap.c

```diff
@@ -66,7 +67,12 @@ int restore_bpfmap_data(int map_fd, uint32_t map_id, struct bpfmap_data_rst **bp
 	void *keys = NULL;
 	void *values = NULL;
 	unsigned int count;
+
+#ifdef LIBBPF_OPTS
+	LIBBPF_OPTS(bpf_map_batch_opts, opts, .elem_flags = 0, .flags = 0, );
```

Comment:
No need to explicitly set fields to `0`, this is done by `LIBBPF_OPTS` (that's the main point of it).